### DISCUSSION TITLE
[Feature] Fallback machine learning endpoint

### DISCUSSION THREAD

**Discussion by aleeraser** [immich-app/immich#13531] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In pair with https://github.com/immich-app/immich/discussions/10685, it would be of great help to use a fallback URL when the remote machine learning endpoint is not reachable for whatever reason (e.g. the computer is off).

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
You can do this with a reverse proxy/loadbalancer such as nginx.
